.github/workflows/macos_intel.yml

```diff
@@ -174,6 +174,12 @@ jobs:
           cd ${HOME}/serialbox
           mkdir build
           cd build
+          XCODE_PATH=$(xcode-select -p)
+          XCODE_HEADERS=${XCODE_PATH}/SDKs/MacOSX.sdk/usr/include
+          XCODE_FRAMEWORK=${XCODE_PATH}/SDKs/MacOSX.sdk/System/Library/Frameworks
+          export C_INCLUDE_PATH+=:${XCODE_PATH}/SDKs/MacOSX.sdk/usr/include
+          export CPLUS_INCLUDE_PATH+=:${XCODE_PATH}/SDKs/MacOSX.sdk/usr/include
+          export LIBRARY_PATH+=:${XCODE_PATH}/SDKs/MacOSX.sdk/usr/lib
```

Comment:
All changes in this file should be removed as the other PR applies the fix.

Reply:
Thats for making the correct fix in PR #17 .